london_cafes/london_cafes/templates/index.html

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
not sure what you mean... 
don't I need set block here to know that this will be the main block on index?